doc/rados/operations/balancer.rst

```diff
@@ -93,43 +102,41 @@ The numeric pool IDs can be gotten with the :command:`ceph osd pool ls detail` c
 Modes
 -----
 
-There are currently two supported balancer modes:
+There are two supported balancer modes:
 
-#. **crush-compat**.  The CRUSH compat mode uses the compat weight-set
-   feature (introduced in Luminous) to manage an alternative set of
-   weights for devices in the CRUSH hierarchy.  The normal weights
-   should remain set to the size of the device to reflect the target
-   amount of data that we want to store on the device.  The balancer
-   then optimizes the weight-set values, adjusting them up or down in
-   small increments, in order to achieve a distribution that matches
-   the target distribution as closely as possible.  (Because PG
-   placement is a pseudorandom process, there is a natural amount of
-   variation in the placement; by optimizing the weights we
-   counter-act that natural variation.)
+#. **crush-compat**. The CRUSH compat mode uses the compat weight-set feature
+   (introduced in Luminous) to manage an alternative set of weights for devices
+   in the CRUSH hierarchy. When the balancer is operating in this mode, the
+   normal weights should remain set to the size of the device in order to
+   reflect the target amount of data intended to be stored on the device. The
+   balancer will then optimize the weight-set values, adjusting them up or down
+   in small increments, in order to achieve a distribution that matches the
+   target distribution as closely as possible. (Because PG placement is a
+   pseudorandom process, it is subject to a natural amount of variation;
+   optimizing the weights serves to counteract that natural variation.)
 
-   Notably, this mode is *fully backwards compatible* with older
-   clients: when an OSDMap and CRUSH map is shared with older clients,
-   we present the optimized weights as the "real" weights.
+   Note that this mode is *fully backward compatible* with older clients: when
+   an OSDMap and a CRUSH map are shared with older clients, Ceph presents the
+   optimized weights as the "real" weights.
 
-   The primary restriction of this mode is that the balancer cannot
-   handle multiple CRUSH hierarchies with different placement rules if
-   the subtrees of the hierarchy share any OSDs.  (This is normally
-   not the case, and is generally not a recommended configuration
-   because it is hard to manage the space utilization on the shared
-   OSDs.)
+   The primary limitation of this mode is that the balancer cannot handle
+   multiple CRUSH hierarchies with different placement rules if the subtrees of
+   the hierarchy share any OSDs. (Such sharing of OSDs is not normal and,
+   because of the difficulty of managing the space utilization on the shared
+   OSDs, is generally not recommended.)
```

Comment:
s/normal/typical/

Reply:
Yeah, that is unambiguously better. Good call.

Reply:
I started off changing `hierarchies` to `roots`, but realized that there are situations where this comes into play under a single root too, sometimes people make fake duplicate hosts, and the way Rook rolls makes the `.mgr` pool problematic until modified.